File: solutions/2219-maximum-number-of-words-found-in-sentences/solution.java
class Solution {
    public int mostWordsFound(String[] sentences) {
        int max = 0;
        for(int i = 0;i<sentences.length;i++){
                int word = sentences[i].split(" ").length;
                if(word > max){
                    max = word;
                }
        }
        return max;
    }
}

File: solutions/2219-maximum-number-of-words-found-in-sentences/solution.cpp
class Solution {
public:
    int mostWordsFound(vector<string>& sentences) {
        int maximum = 0;
        for(string s : sentences){
            int count = 0;
            for(char ch : s){
                if(ch == ' '){
                    count++;
                }
            }
            maximum = max(maximum,count+1);
        }
        return maximum;
    }
};
